src/Arcus.Security.Providers.AzureKeyVault/Extensions/SecretStoreBuilderExtensions.cs

```diff
@@ -384,7 +384,7 @@ public static class SecretStoreBuilderExtensions
         /// <param name="allowCaching">The flag to indicate whether to include caching during secret retrieval in Azure key vault.</param>
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="builder"/> is <c>null</c>.</exception>
         /// <exception cref="ArgumentException">Thrown when the <paramref name="rawVaultUri"/> is blank.</exception>
-        [Obsolete("Use the " + nameof(AddAzureKeyVaultWithManagedIdentity) + " method overload with " + nameof(CacheConfiguration.Default) + " instead to allow default caching")]
+        [Obsolete("Use the " + nameof(AddAzureKeyVaultWithManagedIdentity) + " method overload and pass CacheConfiguration.Default as parameter for the cacheConfiguration to allow default caching")]
```

Comment:
```suggestion
        [Obsolete("Use the " + nameof(AddAzureKeyVaultWithManagedIdentity) + " method overload and pass " + nameof(CacheConfiguration) + "." + nameof(CacheConfiguration.Default) + " as parameter for the cacheConfiguration to allow default caching")]
```

I would be more in favor of doing it this way, to make sure that the docs are always linked to the code.

Reply:
Indeed, that is better.   I didn't think of it to do it like this.